README.md

```diff
@@ -159,4 +161,22 @@ ROSETTA_BENCH_FILTER:STRING=
 To run any of the unittests, execute the relevant test file. For run unittest for filtering:
 ```shell
 python rosetta/test/test_filtering.py
-```
\ No newline at end of file
+```
+
+### (Optional) Run with predefined problem sizes
+- Optionally, you can utilize the `--problemsizefile` argument to specify a predefined problem sizes for the benchmarks. If the `--problemsize` argument is not provided, the default `medium.problemsize.ini` will be used.
+```shell
+python rosetta.py --bench --problemsizefile="benchmarks/mini.problemsize.ini"
+python rosetta.py --bench --problemsizefile="benchmarks/small.problemsize.ini"
+python rosetta.py --bench --problemsizefile="benchmarks/medium.problemsize.ini"
+python rosetta.py --bench --problemsizefile="benchmarks/large.problemsize.ini"
+python rosetta.py --bench --problemsizefile="benchmarks/extralarge.problemsize.ini"
+```
+- Verify the different problem size files excluding CUDA PPM:

```

Comment:
I don't think it is necessary to enumerate all combinations of driver option and problemsizes. Just mention which arguments the `--problemsizefile` takes